docs/dev/publicdashboard/Working_with_public_dashboard.md

```diff
@@ -0,0 +1,131 @@
+ - Note: We use cron jobs to execute the different notebooks in a scheduled way daily to generate the charts.
+ ```
+services:
+    notebook-server:
+        environment:
+        - DB_HOST=db
+        - WEB_SERVER_HOST=0.0.0.0
+        - CRON_MODE=TRUE
+        - STUDY_CONFIG=stage-program
+```
+
+To launch docker-compose.yml:
+```
+docker-compose -f docker-compose.yml up
+```
+
+ Key difference amongst these two docker-compose file is the `CRON_MODE=TRUE` in case of `docker-compose.yml` while its disabled for `docker-compose.dev.yml`.
+
+ # Changes required in docker-compose.dev.yml and/or docker-compose.yml to load different dataset
+
+The default `docker-compose.yml` with DB_HOST=db will load the dataset in MongoDB stored as `Stage_database`.
+
+What if we want to load a different dataset than staging, and test for a different program/study?
+Let us consider the example below: 
+- (Note: This might need to updated once https://github.com/e-mission/e-mission-docs/issues/1048 issue is checked-in.)
+```
+services:
+    notebook-server:
+        environment:
+        - DB_HOST=mongodb://db/openpath_prod_usaid_laos_ev
+        - ....
+        - STUDY_CONFIG=usaid-laos-ev
+```
+1. For testing different config: Change the STUDY_CONFIG, as `STUDY_CONFIG=<program/study type>`.
+ Example: `STUDY_CONFIG=usaid-laos-ev` for the `usaid-laos-ev` study. List of other configs for study/program are accessible from: https://github.com/e-mission/nrel-openpath-deploy-configs/tree/main/configs
+1. Similarly, change the `DB_HOST=<dataset>`.
+Example: `DB_HOST=mongodb://db/openpath_prod_usaid_laos_ev` to load the dataset `openpath_prod_usaid_laos_ev` which has been loaded into the MongoDB.
+
+# Execution of different public dashboard notebooks automatically (without launching Jupyter notebook)
+---
+
+``` 
+$ docker-compose -f docker-compose.yml build
+$ docker-compose -f docker-compose.yml  up 
+$ docker ps
+$ docker exec -it em-public-dashboard-notebook-server-1 /bin/bash
+$ root@06f07def2c0e:/usr/src/app# source setup/activate.sh
+$ (emission) root@06f07def2c0e:/usr/src/app# cd saved-notebooks
+```
+
+Execution of notebooks
+```
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/update_mappings.py mapping_dictionaries.ipynb
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/generate_plots.py generic_metrics.ipynb default
+```
+Refer to additional notebooks from https://github.com/e-mission/em-public-dashboard/blob/main/viz_scripts/docker/crontab
+
+Then go to http://localhost:3274/
+
+- If you want to test for specific configuration (program/study):
+For example, if you want to test for usaid-laos-ev:
+    http://localhost:3274/?study_config=usaid-laos-ev
+and test.
+
+Note: Make sure the Year and Month parameter for the executing notebook is not None. If it's set to None, it will not generate the charts.
```

Comment:
I don't see documentation on changing these to None to run the notebooks.
Also, you should move this up to the setup instructions

Reply:
The documentation for changing this to `None` is mentioned at the initial text of the notebook as 
```
Pass in `None` to remove the filters and plot all data. This is not recommended for production settings, but might be useful for reports based on data snapshots.
```

So, this line is just a cautionary check to remind it's important not to have Year and Month as None. 

This block has been moved before the section where execution of `generate_plots.py` is defined.